docs/Getting Started/Arch Linux/Arch Linux Root on ZFS.rst

```diff
@@ -333,14 +333,7 @@ Create Root and Boot Pools
         -o feature@large_blocks=enabled \
         -o feature@lz4_compress=enabled \
         -o feature@spacemap_histogram=enabled \
-        -O acltype=posixacl \
```

Comment:
Why move these from the top-level to the child dataset?

Reply:
Since these properties are specific for OS, it might be more appropriate to set these properties on the OS dataset only. So I moved them to the child dataset `rpool/sys`

> Setting `-O acltype=posixacl` enables POSIX ACLs globally. If you do not want this, remove that option

Reply:
Moved back.